physionet-django/project/templates/project/published_project.html

```diff
@@ -7,452 +7,462 @@
-            {% endif %}
-          {% elif project.compressed_storage_size %}
-            <li><a href="{% static project.zip_url %}">Download the ZIP file</a> ({{ compressed_size }})</li>
-          {% endif %}
-        {% if data_access %}
-          {% for item in data_access %}
-            {% if item.platform == 1 %} {# ID for AWS open data #}
-              <li>Access using AWS <a href="{{item.location}}">Open Data repository</a></li>
-            {% elif item.platform == 2 %} {# ID for AWS cloud bucket #}
-              <li>Access the data using AWS command line tools: <pre class="shell-command">aws s3 sync {{item.location}} DESTINATION</pre></li>
-            {% elif item.platform == 4 %} {# ID for google BigQuery #}
-              <li><a href="{% url 'published_project_request_access' project.slug project.version 4 %}">Request access</a> using Google BigQuery.</li>
+                                <li><a href="{% static project.zip_url %}">Download the ZIP file</a>
+                                    ({{ compressed_size }})
+                                </li>
+                            {% elif project.compressed_storage_size %}
+                                <li><a href="{% static project.zip_url %}">Download the ZIP file</a>
+                                    ({{ compressed_size }})
+                                </li>
+                            {% endif %}
+                            {% if project.gcp.sent_files %}
+                                <li>Access the files using the Google Cloud Storage Browser <a
+                                        href="https://console.cloud.google.com/storage/browser/{{ project.gcp.bucket_name }}/">here</a>.
+                                    Login with a Google account is required.
+                                <li>Access the data using the Google Cloud command line tools (please refer to the <a
+                                        href="https://cloud.google.com/storage/docs/gsutil_install">gsutil</a>
+                                    documentation for guidance):
+                                    <pre class="shell-command">gsutil -m -u YOUR_PROJECT_ID cp -r gs://{{ project.gcp.bucket_name }} DESTINATION</pre>
+                                </li>
+                            {% endif %}
+                        {% elif project.compressed_storage_size %}
+                            <li><a href="{% static project.zip_url %}">Download the ZIP file</a> ({{ compressed_size }})
+                            </li>
+                        {% endif %}
+                    {% endif %}
+                    {% include "project/published_project_data_access.html" %}
+                    {% if is_wget_supported %}
+                            <li>Download the files using your terminal:
+                                <pre class="shell-command">wget -r -N -c -np {% if project.access_policy %} --user {{ user }} --ask-password {% endif %}
```

Comment:
Something has changed in the formatting of this chunk (though I can't immediately see what!).

Compare current:

![Screenshot 2023-02-08 at 11 49 25 AM](https://user-images.githubusercontent.com/822601/217598431-eabc1ed2-79cd-4dc2-88a4-460ae7803d1e.png)

... with new:

![Screenshot 2023-02-08 at 11 57 29 AM](https://user-images.githubusercontent.com/822601/217598945-f301a454-e708-4716-9da7-581c99fff3c3.png)

Reply:
fixed this as per Benjamin's suggestion